api/comments_api.py

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from datetime import datetime, timezone
```

Comment:
Do you need timezone?

Reply:
No, nor `datetime`! It's a remnant from the way I was previously adding the value for `created`. It's removed now.